### ISSUE TITLE
Transferring the Object Detection function in application to the Server

### ISSUE THREAD

**Issue by noodles8436** [noodles8436/THE-CROSS#4]
The application has low capability as an open-source. 
Thus, the limitation of accessibility of the programming language should be solved by transferring the Object Detection function to the server and using the functionality of Object Detection through network communication.

**Plan : Transferring the Object Detection function to the server**

**Comment by noodles8436** [comment]
Optimized ImageDetector.py and Main.py for Transferring ImageDetector.py to Server
